expression/builtin_encryption.go

```diff
@@ -26,13 +26,16 @@ import (
 	"hash"
 	"io"
 
+	"crypto/aes"
 	"github.com/juju/errors"
 	"github.com/pingcap/tidb/mysql"
 	"github.com/pingcap/tidb/sessionctx"
+	"github.com/pingcap/tidb/sessionctx/variable"
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/util/auth"
 	"github.com/pingcap/tidb/util/chunk"
 	"github.com/pingcap/tidb/util/encrypt"
+	"strings"
```

Comment:
please move `strings` to line 28(it's TiDB rule, standard package need in first import group ^ ^)